thewalrus/internal_modes/useful_tools.py

```diff
@@ -0,0 +1,140 @@
+# Copyright 2019 Xanadu Quantum Technologies Inc.
```

Comment:
maybe worth giving this module a better name.

Reply:
the current one is too useful